db/schema.yml

```diff
@@ -1314,6 +1314,15 @@ firewall_rules:
 - source_ip_range
 - ems_ref
 - network_protocol
+firmwares:
```

Comment:
Is there a simple way extract just these changes, other than winding back this commit, undoing the change and then resubmitting the commit?